src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileFromFileName = new File(fromFileName);
+        File fileToFileName = new File(toFileName);
+        Map<String, Integer> result = new HashMap<>();
+
+        if (!fileFromFileName.exists() || !fileToFileName.exists()) {
```

Comment:
Checking for file existence should be done separately for 'fromFileName' and 'toFileName'. The 'toFileName' should be created if it does not exist, rather than skipping the operation.